dict/portuguese.yml

```diff
@@ -0,0 +1,291 @@
+    'IM': [Ilha de Man, Manês*, Manes*]
+    'IS': [Islândia, Islandês*, Islandes*, Reiquiavique, Reykjavík]
+    'JE': [Ilhas do Canal, Ilhas Anglo-Normandas, Ilhas do Canal*]
+    'LT': [Lituânia, República da Lituânia, Lituan*, Vilnius, Vílnius, Vilna]
+    'LV': [Letónia, Letônia, República da Letônia, Letã*, Letões, Riga]
+    'NO': [Noruega, Reino da Noruega, Norueguês*, Noruegues*, Oslo]
+    'SE': [Suécia, Reino da Suécia, Suec*, Estocolmo]
+    'SJ': [Svalbard e Jan Mayen, Norueguês*, Noruegues*]
+
+  SOUTH:
+    'AD': [Andorra, Principado de Andorra, Principado dos Vales de Andorra, Andorran*, Andorrense*, Andorrian*, Andorrenh*]
+    'AL': [Albânia, República da Albânia, Albanês*, Tirana]
+    'BA': [Bósnia e Herzegovina, Bósnia, Bósni*, Sarajevo, Saraievo]
+    'ES': [Espanha, Reino de Espanha, Reino da Espanha, Espanhol*, Espanhóis, Madrid, Madri, Barcelona]
+    'GI': [Gibraltar, Gibraltarin*, Gibraltin*, Llanit*, Llanis*]
+    'GR': [Grécia, República Helênica, República Helénica, Grego*, Grega*, Atenas]
+    'HR': [Croácia, República da Croácia, Croata*, Zagreb, Zagrebe]
+    'IT': [Itália, República Italiana, Italian*, Roma]
+    'KV': [Kosovo, Cossovo, Cosovo, Kossovo, Kosovar*, Cosovar*, Cossovar*, Pristina]
+    'ME': [Montenegro, Montenegrin*, Podgoritza, Podgoriza, Podgorica]
+    'MK': [Macedônia do NORTH, Macedónia do NORTH, República da Macedônia do NORTH, República da Macedónia do NORTH, Macedóni*, Escópia, Skopje, Skopie]
+    'MT': [Malta, República de Malta, Maltês*, Valeta]
+    'PT': [Portugal, República Portuguesa, Português*, Portugues*, Lisboa]
+    'RS': [Sérvia, República da Sérvia*, Sérvi*, Belgrado]
+    'SI': [Eslovénia, Eslovênia, República da Eslovénia, Esloven*, Liubliana, Ljubljana]
+    'SM': [San Marino, Sereníssima República de San Marino, República de San Marino, São Marinho, São Marino, Samarinês*, São-Marinense*, São-Marinhense*, San-Marinense*]
+    'VA': [Vaticano, Cidade do Vaticano, Estado da Cidade do Vaticano, Vatican*]
+
+  WEST:
+    'AT': [Áustria, República da Áustria, Austríac*, Viena]
+    'BE': [Bélgica, Reino da Bélgica, Belga*, Bélgic*, Bruxelas, Região da Capital Bruxelas]
+    'CH': [Suíça, Confederação Suíça, Suíç*, Zurique, Berna]
+    'DE': [Alemanha, República Federal da Alemanha, Alemã*, Berlim, Frankfurt am Main, Frankfurt sobre o Meno, Francoforte do Meno, Frankfurt, Francoforte]
+    'FR': [França, República Francesa, Francês*, Frances*, Paris]
+    'LI': [Liechtenstein, Listenstaine, Principado de Liechtenstein, Principado do Liechtenstein, Liechtensteinense*, Liechtensteiniense*, Liechtensteinian*, Listenstainian*, Listenstainês*, Vaduz]
+    'LU': [Luxemburgo, Grão-Ducado do Luxemburgo, Grão-Ducado de Luxemburgo, Luxemburguês*, Cidade do Luxemburgo]
+    'MC': [Mónaco, Mônaco, Principado do Mónaco, Principado de Mônaco, Monegasc*]
+    'NL': [Países Baixos, Holanda, Neerlandês*, Holandês*, Amesterdão, Amsterdão, Amsterdã, Amesterdã]
```

Comment:
Is "Amsterdã" correct? Looks "e" is missing.

Reply:
It is correct. It is one of the variations that exist because of the differences between Portuguese from Portugal and from Brazil. Both cases (with and without the "e") are correct in different countries.

Reply:
Very interesting. I was going to ask you about the difference between Portuguese in the two countries. Great to cover both by one dictionary if possible.